src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -1153,6 +1174,12 @@ private class AutoEvoExploringToolWorld
         public readonly Dictionary<OrganelleDefinition, (double Percentage, double Average)>
             MicrobeSpeciesOrganelleStatistics = new();
 
+        /// <summary>
+        ///   Used to generate world statistics
```

Comment:
Shouldn't this clarify that this is for calculating world *upgrade* statistics? To me this comment like it is right now doesn't really seem to match the variable this is on...